### ISSUE TITLE
Mobs standing back don't face their target

### ISSUE THREAD

**Issue by teschnei** [DarkstarProject/darkstar#3380]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:**

**_Server Version_** (type `@revision` in game) **:**
c6a776219f23d9bdb6c584dc5d95e8c214c85cd8

**_Source Branch_** (master/stable) **:**
master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:**

Even when using a ranged attack, they won't face their target

**Comment by abriasffxi** [comment]
I was going to make an issue, but not sure if this is the same deal.

When a mob uses a tp move, they do not face their target first.  So, for example, a Damselfly using Venom that sees an aggro change can "target" a player on one side, but their model remains facing the player that had aggro previously.  And, the TP Move then hits both the frontal cone of the model  and the player that it is targeted.

Same behavior when a mob uses a spell, although the mechanics aren't as bad.  And, as you noted, ranged attacks are the same.

We're just now starting dragons, but I see this as a pretty big potential issue with Tail Swipe behavior etc.  It's also a problem for SA/TA.

One thing I'm not sure about - when a mob is bound, they do not rotate to face the player.  I thought I remembered that they could rotate when bound.

**Comment by teschnei** [comment]
Clarifications: a ranged attack is a tp move, so they are the same issue.

A dragon does not turn either way, so it won't make a difference.

I've already fixed not turning when standing back locally, but they still did not turn before firing ranged attacks (and probably tp moves) or spells, so I am waiting for that to be fixed before I push any of it.  I did not know about not turning when bound though, I will make sure I look into that before closing this.
